readme.md

```diff
@@ -450,6 +450,7 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 	- [Iridium](https://github.com/SierraSoftworks/Iridium) - A high performance MongoDB ORM with support for promises, distributed caching, preprocessing, validation and plugins.
 	- [OpenRecord](https://github.com/PhilWaldmann/openrecord) - ORM for PostgreSQL, MySQL, SQLite3 and RESTful datastores. Similar to ActiveRecord.
 	- [orm2](https://github.com/dresende/node-orm2) - ORM for PostgreSQL, MariaDB, MySQL, Amazon Redshift, SQLite, MongoDB.
+	- [firenze](https://github.com/fahad19/firenze) - Adapter based ORM for server and the browser. Supports MySQL, Memory, Redis, localStorage and more.
```

Comment:
Should it be `adapter-based`? I also don't think the server/browser mention is relevant seeing how this list is for Node.js.

How about:

> Adapter-based ORM for MySQL, Memory, Redis, localStorage and more.